ios/core/Sources/Types/Generic/AnyType.swift

```diff
@@ -154,6 +167,12 @@ struct CustomEncodable: CodingKey {
         self.stringValue = key
         if let encodable = encodable as? Encodable {
             self.data = encodable
+        } else if
+            let encodable,
+            let data = try? JSONSerialization.data(withJSONObject: encodable, options: .fragmentsAllowed),
```

Comment:
`.fragmentsAllowed` lets us encode `String` here as a top level type, as we dont know what the value type will be